README.md

```diff
@@ -179,5 +179,135 @@ binned_aligned_spikes = BinnedAlignedSpikes(
 
 As with the previous example this can be then added to a processing module in an NWB file and written to disk using exactly the same code as before.
 
+### Storing data from multiple events together
+In experiments where multiple stimuli are presented to a subject within a single session, it is often useful to store the aggregated spike counts from all events in a single object. For such cases, the `AggregatedBinnedAlignedSpikes` object is ideal. This object functions similarly to the `BinnedAlignedSpikes` object but is designed to store data from multiple events (e.g., different stimuli) together.
+
+Since events may not occur the same number of times, an homogeneous data structure is not possible. Therefore the `AggregatedBinnedAlignedSpikes` object includes an additional variable, event_indices, to indicate which event each set of counts corresponds to. You can create this object as follows:
+
+
+```python
+from ndx_binned_spikes import AggregatedBinnedAlignedSpikes
+
+aggregated_binned_aligned_spikes = AggregatedBinnedAlignedSpikes(
+    bin_width_in_milliseconds=bin_width_in_milliseconds,
+    milliseconds_from_event_to_first_bin=milliseconds_from_event_to_first_bin,
+    data=data,  # Shape (number_of_units, aggregated_events_counts, number_of_bins)
+    timestamps=timestamps,  # As many timestamps as the second dimension of data
+    event_indices=event_indices,  # An index indicating which event each of the counts corresponds to
+)
+```
+
+The `aggregated_events_counts` represents the total number of repetitions for all the events being aggregated. For example, if data is being aggregated from two stimuli where the first stimulus appeared twice and the second appeared three times, the aggregated_events_counts would be 5.
+
+The `event_indices` is an indicator vector that should be constructed so that `data[:, event_indices == event_index, :]` corresponds to the binned spike counts around the event with the specified event_index. You can retrieve the same data using the convenience method `aggregated_binned_aligned_spikes.get_data_for_event(event_index)`.
```

Comment:
Try not to confuse the terms "event" and "stimulus". You have a stimulus index, not an even stimulus.

Reply:
Are you thinking that this structure is only for aggregating stimulus? I think that would simplify the language but the other structure in the extension is not only for stimulus but for events, trials, etc.